src/next/blocks/blockRepositoryBranchProtection.ts

```diff
@@ -16,7 +16,7 @@ export const blockRepositoryBranchProtection = base.createBlock({
 					id: "branch-protection",
 					async send({ octokit }) {
 						await octokit.request(
-							`PUT /repos/{owner}/{repository}/branches/{main}/protection`,
+							`PUT /repos/{owner}/{repo}/branches/{branch}/protection`,
```

Comment:
Now that I set this correctly, the types later on are saying I shouldn't use `checks` the way I was. But per #1309 I'm going to switch to a ruleset anyway.